tests/rptest/tests/metrics_reporter_test.py

```diff
@@ -134,6 +147,9 @@ def assert_fields_are_the_same(metadata, field):
         assert last["config"]["log_message_timestamp_type"] == "CreateTime"
         assert last["redpanda_environment"] == "test"
 
+        last_post_restart = metadata.pop()
+        assert last_post_restart["id_hash"] == last["id_hash"]
```

Comment:
This is a nice PR, you could go the extra mile and also just obtain the checksum of the raw license in python and compare it to what redpanda returns.

Reply:
Might be as easy as:
```
>>> import hashlib
>>> hashlib.md5(sample_license()).hexdigest()
'2a53375ff139d9837e93a38a279d63e5'
```

Reply:
Nice idea, I made the change.